README.md

```diff
@@ -50,8 +50,9 @@ brew untap chanzuckerberg/fogg
 Binaries are available on the releases page. Download one for your architecture, put it in your path and make it executable.
 
 Instructions on downloading the binary:
-1. Go here: https://github.com/chanzuckerberg/fogg/releases, go to the release you want to download, find the tar file that is correct for your machine, and right click the link. Then click "Copy Link Address".
-2. Run `curl -s https://raw.githubusercontent.com/chanzuckerberg/fogg/master/download.sh | bash -s -- -b PATH VERSION` where PATH is where you want to install and VERSION is the version you want to install. To find the path of your current fogg, you can run `which fogg`. Then use the path that is outputted as PATH in the command.
+1. Go here: https://github.com/chanzuckerberg/fogg/releases to find which version of fogg you want.
+2. Run `rm -r PATH; curl -s https://raw.githubusercontent.com/chanzuckerberg/fogg/master/download.sh | bash -s -- -b PATH-FOLDER VERSION` where PATH is where you want to install fogg and VERSION is the specific release version you want to install (format is vx.yy.z). To find the path of your current fogg, you can run `which fogg`. Then use the path that is outputted as PATH in the command. The PATH-FOLDER is the folder in which fogg will be installed.
```

Comment:
`rm -r PATH;` why is that necessary? will the download script fail if it finds something there? I think naming this `PATH` is somewhat confusing (and I realize that is how it was before) because when I see `PATH` in some shell script I will first think about the PATH environment variable.

Reply:
ya it will fail if something is there

Reply:
@edulop91 can we merge this?